integration_tests/suite/test_cdr.py

```diff
@@ -252,7 +307,12 @@ def test_given_no_call_logs_when_list_cdr_then_empty_list(self):
          'destination_name': '.tsèd',
          'direction': 'outbound',
          'source_exten': '7867',
-         'source_name': '.rùos'},
+         'source_name': '.rùos',
+         'participants': [{'user_uuid': USER_1_UUID,
+                           'tenant_uuid': VALID_TENANT,
+                           'line_id': '1',
+                           'tags': ['rh', 'Poudlard'],
+                           'role': 'destination'}]},
```

Comment:
Why you modify the test to add a participant? A call log can exist without having a participant. If your code depend on participants, then you have a problem

Reply:
Yes it can exists, but since it's not belong your tenant you can't get it via API.

Reply:
Yes we MUST get it via API. Probably linked to the master tenant?

Reply:
We have found ton of contexts and associated tenant with them.